fidesops.toml

```diff
@@ -41,6 +41,7 @@ worker_enabled = false
 
 [root_user]
 analytics_opt_out = false
+analytics_id = "bc03820cc476fd0bb87322950f07b295"
```

Comment:
We'll need to remove this

Reply:
I ran lots of the `nox` commands on this one. Unfortunately I was watching for this and still somehow missed it so I'm not sure which one triggered it.